modules/programs/wpaperd.nix

```
@@ -0,0 +1,52 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.wpaperd;
+  tomlFormat = pkgs.formats.toml { };
+in {
+  meta.maintainers = [ maintainers.Avimitin ];
```

Like you said, you need to be in the list of either Nixpkgs or Home Manager maintainers. I don't see you in the Nixpkgs list so please add yourself to `modules/lib/maintainers.nix` and change this line to

```suggestion
  meta.maintainers = [ hm.maintainers.Avimitin ];
```